### PR TITLE
curielogger: cleanup duplicated fields

### PR DESCRIPTION
**Pull request by flaper87**
Signed-off-by: Flavio Percoco <flavio@reblaze.com>

### CODE REVIEW

File: e2e/test_e2e.py [hunk 1]
```diff
@@ -162,26 +162,13 @@ def target(request):
...
-                "statement": (
-                    "SELECT Path FROM logs " "ORDER BY StartTime DESC LIMIT 1024"
-                ),
-                "parameters": [],
-            }
-            res = requests.post(self._base_url + "/logs/api/v1/exec/", json=data)
-            for log in res.json():
-                if pattern in log[0]:
-                    return True
-        else:
```
Comment (flaper87) [inline e2e/test_e2e.py:179]: @xavier-rbz Do you know/remember why this was needed? I would rather just use ES' DSL if this SQL query is not required :)

Comment (Aviv-Galmidi) [reply e2e/test_e2e.py:179]: The path "/logs/api/v1/exec/" was the old API path we used for requesting logs before we switched over to ELK and it is no longer in use, if that helps you in any way :)

File: curiefense/curielogger/curielogger/main.go [hunk 2]
```diff
@@ -171,9 +165,10 @@ func (s grpcServerParams) StreamAccessLogs(x als.AccessLogService_StreamAccessLo
 					Version:     tls.GetTlsVersion().String(),
 				},
 				Request: RequestData{
+					RequestId:    req.GetRequestId(),
+					Scheme:       req.GetScheme(),
 					BodyBytes:    req.GetRequestBodyBytes(),
 					HeadersBytes: req.GetRequestHeadersBytes(),
-					OriginalPath: req.GetOriginalPath(),
```
Comment (xavier-rbz) [inline curiefense/curielogger/curielogger/main.go:176]: shouldn't `OriginalPath` be removed from common.go?

File: curiefense/curielogger/curielogger/common.go [hunk 3]
```diff
@@ -123,13 +125,8 @@ type MetadataData struct {
 }
 
 type CuriefenseLog struct {
-	RequestId string `json:"requestid"`
 	Timestamp string `json:"timestamp"`
-	Scheme    string `json:"scheme"`
 	Authority string `json:"authority"`
-	Port      uint32 `json:"port"`
-	Method    string `json:"method"`
-	Path      string `json:"path"`
```
Comment (xavier-rbz) [inline curiefense/curielogger/curielogger/common.go:130]: May I suggest updating `es_index_template.json` to reflect these changes?

**Review by xavier-rbz — CHANGES_REQUESTED** [review]

File: curiefense/curielogger/curielogger/common.go (hunk 3 above)
Comment (flaper87) [reply curiefense/curielogger/curielogger/common.go:130]: > May I suggest updating es_index_template.json to reflect these changes?

Oh, man! I can't believe I forgot to do this :facepalm:

**Review by xavier-rbz — APPROVED** [review]

**Pull request merged**